apps/test/unit/lib/levelbuilder/code-docs-editor/OrderableListTest.jsx

```diff
@@ -39,9 +39,12 @@ describe('OrderableList', () => {
     const wrapper = shallow(<OrderableList {...defaultProps} />);
     wrapper
       .find('.fa-trash')
-      .at(1)
+      .at('1')
```

Comment:
Should this at be changing to `'1'` too?

Reply:
Gotta love JS allowing this to still work though